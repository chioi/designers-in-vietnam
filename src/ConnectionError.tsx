import React from "react";

const ConnectionError = () => (
  <div>
    <p>Something went wrong :-(</p>
    <p>Make sure you have internet connection</p>
  </div>
);

export default ConnectionError;
