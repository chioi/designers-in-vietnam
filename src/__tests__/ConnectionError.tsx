import React from "react";
import { create } from "react-test-renderer";
import ConnectionError from "../ConnectionError";

describe("gets created", () => {
  it("matches the snapshot", () => {
    expect(create(<ConnectionError />)).toMatchSnapshot();
  });
});
