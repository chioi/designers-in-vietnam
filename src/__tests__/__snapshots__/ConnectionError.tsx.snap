// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets created matches the snapshot 1`] = `
<div>
  <p>
    Something went wrong :-(
  </p>
  <p>
    Make sure you have internet connection
  </p>
</div>
`;
